src/Framework/Bootloader/Http/RouterBootloader.php

```diff
@@ -107,6 +107,13 @@ private function router(
         );
     }
 
+    private function handler(?CoreInterface $core = null, ContainerInterface $container): HandlerInterface
```

Comment:
This will fix a fatal error:

```suggestion
    private function handler(
        #[Proxy] ContainerInterface $container,
        ?CoreInterface $core = null,
    ): HandlerInterface {
```